spec/utils_spec.rb

```diff
@@ -145,6 +145,13 @@
             .and empty_options
   end
 
+  it "should use sha256 for a signature when long_url_signature is true" do
+    expected_signature = Cloudinary::Utils.cloudinary_url("test", :format => "jpg", :sign_url => true, :long_url_signature => true).match(/s--[0-9A-Za-z_-]{32}--/).to_s
```

Comment:
there is not ground truth in this test. if hashing algorithm is changed/broken, this test will still pass.

Please see the reference implementation.